### ISSUE TITLE
First-run wizard breaks and renders NextCloud unusable when video autoplay is disabled

### ISSUE THREAD

**Issue by deviant** [nextcloud/firstrunwizard#1323]
### Steps to reproduce
1. Ensure that audio/video autoplay are disabled
2. Open NextCloud

### Expected behaviour
NextCloud can be used

### Actual behaviour
A blank white rectangle on a dark transparent background appears, blocking out the UI. It is impossible to dismiss this, and NextCloud cannot be used normally any more. Turning on audio/video (not sure which of these is important, probably video?) autoplay and refreshing displays the first-run wizard and allows me to click through and close it. I would attach a screenshot, except I don't know how to trigger the first-run wizard again.

This isn't a problem with how my browser is configured; NextCloud should not become bricked when videos cannot be autoplayed just because it wants to display a fancy animation to new users (or users who have just upgraded NextCloud, as in my case). It should fall back gracefully and display a static graphic. For new users, this is obviously a pretty terrible user experience.

### Server configuration
N/A

### Client configuration
**Browser:** Firefox 129.0.1
**Operating system:** N/A

### Logs
N/A

#### Nextcloud log (data/owncloud.log)
N/A

#### Browser log
Don't know how to retrigger the first-run wizard, so can't get these.

**Comment by lolcabanon** [comment]
Hey! I am not exactly sure where to report, so I will do this here. Just tell me if I should open a separate issue elsewhere.

I just upgraded to NC31, and it seems a couple of links/redirects are broken.

This might be a server config issue, but this is really not my expertise, so I wouldn't really know...

## From the JS console :

>GET https://nc.example.com/index.php/settings/api/apps/media?fileName=https%3A%2F%2Fnextcloud.com%2Fc%2Fuploads%2F2025%2F02%2FNextcloud-Hub-10-featured-image.png
500 (Internal Server Error)
>
>GET https://nc.example.com/index.php/settings/api/apps/media?fileName=https%3A%2F%2Fnextcloud.com%2Fc%2Fuploads%2F2024%2F10%2FNextcloud-podcast-Open-source-government-Schleswig-Holstein.png
500 (Internal Server Error)
>
>GET https://nc.example.com/index.php/settings/api/apps/media?fileName=https%3A%2F%2Fgithub.com%2Fnextcloud%2Ffirstrunwizard%2Fraw%2Fmaster%2Fimg%2FNextcloud.webm
500 (Internal Server Error)

## With `'debug' => true` and [this link](https://nc.example.com/index.php/settings/api/apps/media?fileName=https%3A%2F%2Fnextcloud.com%2Fc%2Fuploads%2F2025%2F02%2FNextcloud-Hub-10-featured-image.png) :

```
Erreur interne du serveur
Le serveur est incapable d'exécuter votre requête.

Si cela se reproduit, veuillez envoyer les détails techniques ci-dessous à l'administrateur du serveur.

Le fichier journal du serveur peut fournir plus de renseignements.

Renseignements techniques
Adresse distante : 184.162.69.58
ID de la demande : Z9znpW56YIMJJCtiF-mcuQAAAVY
Type : Exception
Code : 0
Message : OCA\Settings\Controller\AppSettingsController::getAppDiscoverMedia(): Argument #1 ($fileName) must be of type string, null given, called in /home/user/public_html/lib/private/AppFramework/Http/Dispatcher.php on line 200 in file '/home/user/public_html/apps/settings/lib/Controller/AppSettingsController.php' line 131
Fichier : /home/user/public_html/lib/private/AppFramework/Http/Dispatcher.php
Ligne : 146

Trace
#0 /home/user/public_html/lib/private/AppFramework/App.php(161): OC\AppFramework\Http\Dispatcher->dispatch()
#1 /home/user/public_html/lib/private/Route/Router.php(307): OC\AppFramework\App::main()
#2 /home/user/public_html/lib/base.php(1025): OC\Route\Router->match()
#3 /home/user/public_html/index.php(24): OC::handleRequest()
#4 {main}

Précédent
#0 /home/user/public_html/lib/private/AppFramework/Http/Dispatcher.php(200): OCA\Settings\Controller\AppSettingsController->getAppDiscoverMedia()
#1 /home/user/public_html/lib/private/AppFramework/Http/Dispatcher.php(114): OC\AppFramework\Http\Dispatcher->executeController()
#2 /home/user/public_html/lib/private/AppFramework/App.php(161): OC\AppFramework\Http\Dispatcher->dispatch()
#3 /home/user/public_html/lib/private/Route/Router.php(307): OC\AppFramework\App::main()
#4 /home/user/public_html/lib/base.php(1025): OC\Route\Router->match()
#5 /home/user/public_html/index.php(24): OC::handleRequest()
#6 {main}
```

**Comment by joshtrichards** [comment]
@lolcabanon This seems unrelated. Please troubleshoot your situation on the forum - https://help.nextcloud.com

**Comment by joshtrichards** [comment]
There appears to be no widely supported means of detecting whether autoplay is functioning, but I think we can implement *Example 3* (most widely supported) from MDN in parallel with *Example 2* (presumably what will be the main way of doing so in the future):

https://developer.mozilla.org/en-US/docs/Web/Media/Guides/Autoplay#autoplay_availability

https://developer.mozilla.org/en-US/docs/Web/Media/Guides/Autoplay#the_autoplay_attribute

https://github.com/nextcloud/firstrunwizard/blob/master/src/components/pages/IntroAnimation.vue

**Comment by MrDanielHarka** [comment]
This is not cool. I had the same issue that OP mentioned on a clean NextCloud install.
I had "Autoplay" set to "Default (Allow)" in Vivaldi Browser settings and had to explicitly set it to "Allow".
This is a very basic and stupid mistake on NextCloud's side.

<img width="332" height="103" alt="Image" src="https://github.com/user-attachments/assets/e2214fa8-6bcc-4e57-a08d-06183ebe3bf1" />
